ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -62,7 +62,7 @@ RUN pip3 install -U setuptools
 @[if ros_version == 2]@
 RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y ros-@(rosdistro_name)-ros-workspace
 @[end if]@
-RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y ccache
+RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y ccache python3-catkin-pkg-modules
```

Comment:
Should `python3-catkin-pkg-modules` be moved to the apt invocation below so that it's only installed when needed for the ABI checker? That might also reduce the need to install it for some of the travis tests.

Reply:
It was also a good idea to me. After implemented it, travis pointed me that [we are including the abi-checker code unconditionally inside the build_and_install.py](https://github.com/ros-infrastructure/ros_buildfarm/pull/681/files#diff-14a2d9d1f397cfe07f871fdb61dc0483R22) file.

Reply:
Is this a situation where it is worth moving the include into the function definition? catkin-pkg is not a heavyweight dependency per se but it wasn't needed before and is clearly creating some churn in our tests.

Reply:
Not a bad idea. See 6e92b5b054db132ec0be3cb2f058415b38b0bf8a , 43528c579d6ee045adcc3b00cc25a45e5513c052 and 955ba5bb1ca93b013e32b42edca6480b847ec404